zrml/neo-swaps/src/benchmarking.rs

```diff
@@ -73,6 +74,14 @@ fn create_market<T: Config>(
     maybe_market_id.unwrap()
 }
 
+fn create_spot_prices<T: Config>(asset_count: u16) -> Vec<BalanceOf<T>> {
+    let mut result = vec![MIN_SPOT_PRICE.saturated_into(); (asset_count - 1) as usize];
+    let remaining_u128 =
+        ZeitgeistBase::<u128>::get().unwrap() - (asset_count - 1) as u128 * MIN_SPOT_PRICE;
+    result.push(remaining_u128.saturated_into());
+    result
+}
```

Comment:
b11f80cd128a7c7f11e7c16a0a15adf38ad7b2c5

Doing equal prices feels weirder to me than doing this because you're most often forced to handle a remainder (for example, 1/3 is not a fixed point number).